website/static/js/projectorganizer.js

```diff
@@ -1234,15 +1366,23 @@ var tbOptions = {
     },
     onload : function () {
         var tb = this,
-            rowDiv = $('.tb-row');
+            rowDiv = tb.select('.tb-row');
         _poLoadOpenChildren.call(tb);
-       rowDiv.first().trigger('click');
+        rowDiv.first().trigger('click');
 
-        $('.gridWrapper').on('mouseout', function(){
+        $('.gridWrapper').on('mouseout', function () {
             rowDiv.removeClass('po-hover');
         });
-
-
+        $(document).on('keypress', '#renameInput', function () {
```

Comment:
I still think adding this event handler to the element, together with the `onKey` pattern I suggested, would be an improvement for consistency and cohesion.